pyproject.toml

```diff
@@ -61,7 +61,7 @@ doc = [
     "docutils==0.16",
 ]
 speculos = [
-    "speculos>=0.5.0",
```

Comment:
Last speculos version is something like 0.7.1, why not using it?

Reply:
I can change it, but I was thinking the upgrade is not mandatory as it doesnt break compatibility